packages/dds/tree/src/jsonDomainSchema.ts

```diff
@@ -14,97 +14,115 @@ import {
 const sf = new SchemaFactory("com.fluidframework.json");
 
 /**
- * {@link AllowedTypes} for primitives types allowed in JSON.
- * @internal
+ * Utilities for using storing JSON data in {@link TreeNode}s.
+ * @remarks
+ * Schema which replicate the JSON data model with {@link TreeNode}s.
+ *
+ * This allows JSON to be losslessly round-tripped through a tree with the following limitations:
+ *
+ * 1. Only information that would be preserved by JSON.parse is preserved. This means (among other things) that numbers are limited to JavasScript's numeric precision.
+ * 2. The order of fields on an object is not preserved. The resulting order is arbitrary.
+ *
+ * JSON data can be imported from JSON into this format using `JSON.parse` then {@link TreeAlpha.importConcise} with the {@link JsonAsTree.(Tree:variable)} schema.
+ *
+ * @alpha
  */
-export const JsonPrimitive = [
-	sf.null,
-	sf.boolean,
-	sf.number,
-	sf.string,
-] as const satisfies AllowedTypes;
+export namespace JsonAsTree {
+	/**
+	 * {@link AllowedTypes} for primitives types allowed in JSON.
+	 * @alpha
+	 */
+	export const Primitive = [
+		sf.null,
+		sf.boolean,
+		sf.number,
+		sf.string,
+	] as const satisfies AllowedTypes;
 
-/**
- * @internal
- */
-export type JsonPrimitive = TreeNodeFromImplicitAllowedTypes<typeof JsonPrimitive>;
+	/**
+	 * @alpha
```

Comment:
What is the current behavior if someone incorrectly tags an `@alpha` namespace member as `@internal`?